rcaracaus/pledge-vegan-node
# found a better way to authenticate for clear-cache

found a way to let the server handle authentication for the web page and leave nginx out of it

Steps to test:
1. nodemon app.js
2. npm install new modules
3. Go to localhost:3000/clear-cache
4. Use appropriate credentials
5. Check that the page authenticates and allows you to clear cache
